experimental/nodejs-functions/README.md

```diff
@@ -64,4 +68,15 @@ module.exports.get = function(req, res, next) {
 
 If you wish to have multiple functions as part of your project, simply create an additonal `.js` file in your folder to contain the function.
 
-Note that you can also `require` any modules that you wish to use but adding them to the `package.json` and adding a `require` statement into your function files as you would normally.
\ No newline at end of file
+Note that you can also `require` any modules that you wish to use but adding them to the `package.json` and adding a `require` statement into your function files as you would normally.
+
+## Configuring Experimental Repo
+
+Upgrade your CLI to the latest version and add the experimental repo:
+```bash
+1. brew upgrade appsody
+
+2. appsody repo add experimental https://github.com/appsody/stacks/releases/latest/download/experimental-index.yaml
+
+3. appsody init experimental/nodejs-functions
```

Comment:
perhaps init does not need to be here.. you can add a line to say refer to tracks like <repo>/<stack-id>